core/lib/devise/strategies/remote_authenticatable.rb

```diff
@@ -13,7 +13,7 @@ def authenticate!
         auth_params = params[scope]
         
         # mapping.to is a wrapper over the resource model
-        resource = mapping.to.new
+        resource = mapping.to.where(email: auth_params[:email]).first || mapping.to.new
```

Comment:
```suggestion
        resource = mapping.to.find_or_initialize_by(email: auth_params[:email])
```
?

Reply:
Can't use `find_or_initialize_by` here since it's not an active record model. The request for the user is being sent to mnohub.

Reply:
by all means / disregard then :)